app/app.module.js

```diff
@@ -125,77 +124,13 @@ var loadStylesheet = function (url) {
 // todo: load json from rest endpoint
 $.get('static/conf/plugins.json', function (plugins) {
 
-  ENTERPRISE_PLUGINS = plugins.filter(function (p) {
-    return p.enterprise;
-  }).map(function (el) {
-    return {
-      name: el.name,
-      files: [el.uri],
-      routing: el.routing,
-      stylesheet: el.stylesheet
-    };
-  });
-  DEFAULT_PLUGINS = plugins.filter(function (p) {
-    return !p.enterprise;
+  PLUGINS = plugins.filter(function (p) {
+    return p.enabled;
```

Comment:
As all plugins are enabled by default now, it looks like we could simply remove this feature and just list all plugins?
Afaik this whole filtered-plugin feature was only reasonable by enterprise plugin distinction..

Reply:
I also tried to think about it and wanted to keep the filtering code for... reasons [1]. That's why I amended it to filter by and `enabled` configuration attribute. However, we can also remove this logic altogether.

[1] Maybe someone will come up with plugins we only want to enable for specific builds? However, I recognize this is completely hypothetical, just wanted to share my thoughts.

Reply:
I'd remove it completely, we can re-add any time when there is a real need for.

Reply:
This has been addressed, thanks.